meteor.js

```diff
@@ -1,244 +1,318 @@
-
-
-            if(settingsJson.public)
-                settings.PUBLIC_SETTINGS = settingsJson.public;
-
-            scripts = scripts.replace('__meteor_runtime_config__', '<script type="text/javascript">__meteor_runtime_config__ = JSON.parse(decodeURIComponent("'+encodeURIComponent(JSON.stringify(settings))+'"));</script>');
-            
-            // add Meteor.disconnect() when no server is given
-            if(!program.url)
-                scripts += "\n"+'        <script type="text/javascript">Meteor.disconnect();</script>';
-
-            content = content.replace(/{{ *> *scripts *}}/, scripts);
-
-            // write the index.html
-            return fs.writeFileAsync(path.join(buildPath, 'index.html'), content);
-        });
-    },
-    cleanUp: function(callback) {
-        return Q.try(function() {
-            // remove files
-            deleteFolderRecursive(path.join(buildPath, 'bundle'));
-            fs.unlinkSync(path.join(buildPath, 'program.json'));
-            try{
-                fs.unlinkSync(path.join(buildPath, 'head.html'));
-            } catch (e){
-                console.log("Didn't unlink head.html; doesn't exist.");
-            }            
+      // ADD HEAD
+      content = content.replace(/{{ *> *head *}}/, head);
+      content = content.replace(/{{ *> *body *}}/, body);
+
+      // get the css and js files
+      const files = {
+        css: [],
+        js: []
+      };
+
+      _.each(fs.readdirSync(outputPath), (file) => {
+        if (/^[a-z0-9]{40}\.js$/.test(file)) {
+          files.js.push(file);
+        } else if (/^[a-z0-9]{40}\.css$/.test(file)) {
+          files.css.push(file);
+        }
+      });
+
+      let primaryCSSfile = files.css[0];
+
+      // --debug case
+      if (program.debug) {
+        const json = fs.readFileSync(path.resolve(path.join(outputPath, 'program.json')), {encoding: 'utf-8'});
+        const prog = JSON.parse(json);
+
+        _.each(prog.manifest, (item) => {
+          if (item.type === 'js' && item.url) {
+            files.js.push(item.path.replace(/^app\//, '') + '?hash=' + item.hash);
+          } else if (item.type === 'css' && item.url) {
+            // for css file cases, do not append hash.
+            files.css.push(item.path.replace(/^app\//, ''));
+
+            if (item.url.includes('meteor_css_resource=true')) {
+              primaryCSSfile = item.path.replace(/^app\//, '');
+            }
+          }
         });
-    }
-}
+
+        print('[DEBUG] Files:', files, {primaryCSSfile});
+      }
+
+      // MAKE PATHS ABSOLUTE
+      if(_.isString(program.path)) {
+        // fix paths in the CSS file
+        if(!_.isEmpty(files.css)) {
+          _.each(files.css, (css, i) =>  {
+            var cssFile = fs.readFileSync(path.join(outputPath, css), { encoding: 'utf8' });
+            cssFile = cssFile.replace(/url\(\'\//g, `url('${program.path}`).replace(/url\(\//g, `url(${program.path}`);
```

Comment:
Also here, maybe putting RegEx into a const?

Reply:
> Maybe putting the regex into a const in higher level scope?

Global RegExp can't be static